code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityMapTests.java

```diff
@@ -310,6 +310,83 @@ public void test_FromData() throws Exception {
 		assertEquals("false", flattenedMap.get("identityMap.USERID[0].primary"));
 	}
 
+	@Test
+	public void test_FromData_removesItem_emptyId() throws Exception {
```

Comment:
Are we also covering the use-case where there may be previously stored ids with empty id value, so that the SDK clears them and stores the updated identity map? And if so, do we need a test for this scenario?